package.json

```
@@ -13,7 +13,7 @@
     "preeslint": "pnpm --filter '@openverse/eslint-plugin' run build",
     "eslint": "eslint --ext .js,.ts,.vue,.json,.json5 --ignore-path .gitignore --ignore-path .eslintignore --max-warnings=0 frontend automations/js packages .pnpmfile.cjs .eslintrc.js prettier.config.js tsconfig.base.json"
   },
-  "packageManager": "pnpm@8.15.6",
+  "packageManager": "pnpm@8.15.7",
   "engines": {
     "node": ">= 18.0.0 <19",
     "pnpm": ">= 8.15.6"
```

Also update `engines.pnpm`.
```suggestion
    "pnpm": ">= 8.15.7"
```